src/resources/views/list.blade.php

```diff
@@ -178,7 +178,7 @@
       @endif
 
 	  	var table = $("#crudTable").DataTable({
-        "pageLength": {{ $crud->getDefaultPageLength() }},
+        "pageLength": "{{ $crud->getDefaultPageLength() }}",
```

Comment:
Page length is a number - should numbers be surrounded by quotes?

Reply:
You're right - thank you, fixed!